src/components/vistas/foundations-selector/index.js

```diff
@@ -14,8 +15,8 @@ function FoundationsSelector() {
       <div className={styles.contenedorPrincipal}>
         <Searcher placeholder="¿En qué fundación querés retirar?" />
         <ContainerFundaciones title="FUNDACIONES AMIGAS" />
-        <Link to="/scanning">
-          {/*ACA VA EL COMPONENTE QUE LE SIGUE, HABRIA QUE IMPORTARLO EN LA LINEA 8*/}
+        <Link to="/scanner">
+          {Scanner}

```

Comment:
Preferir notación en JSX.
```suggestion
          <Scanner />
```

Reply:
No puedo rutear si escribo notacion JSX